plugins/inputs/prometheus/sample.conf

```diff
@@ -2,46 +2,70 @@
 [[inputs.prometheus]]
   ## An array of urls to scrape metrics from.
   urls = ["http://localhost:9100/metrics"]
-  
+
   ## Metric version controls the mapping from Prometheus metrics into Telegraf metrics.
   ## See "Metric Format Configuration" in plugins/inputs/prometheus/README.md for details.
   ## Valid options: 1, 2
   # metric_version = 1
-  
+
   ## Url tag name (tag containing scrapped url. optional, default is "url")
   # url_tag = "url"
-  
+
   ## Whether the timestamp of the scraped metrics will be ignored.
   ## If set to true, the gather time will be used.
   # ignore_timestamp = false
-  
+
   ## An array of Kubernetes services to scrape metrics from.
   # kubernetes_services = ["http://my-service-dns.my-namespace:9100/metrics"]
-  
+
   ## Kubernetes config file to create client from.
   # kube_config = "/path/to/kubernetes.config"
-  
-  ## Scrape Kubernetes pods for the following prometheus annotations:
-  ## - prometheus.io/scrape: Enable scraping for this pod
-  ## - prometheus.io/scheme: If the metrics endpoint is secured then you will need to
-  ##     set this to 'https' & most likely set the tls config.
-  ## - prometheus.io/path: If the metrics path is not /metrics, define it with this annotation.
+
+  ## Explicit scrape configuration for pods matched by label, field and
+  ## namespace selectors. Values cannot be overridden by the `prometheus.io/`
+  ## pod annotations. Not specified or enabled = false, then scraping config is
```

Comment:
this section looks out of place

Reply:
ah agreed, does it make sense to merge this into the "Scrape 'annotations' method settings" section?

Reply:
IDK, just spotted something odd. As I said, feel free to take over this PR :)